ee/clickhouse/sql/person.py

```diff
@@ -57,6 +58,7 @@
 properties,
 is_identified,
 is_deleted,
+version,
 _timestamp,
```

Comment:
While at it, can we add `_timestamp_ms` to this table and the events one? The `_timestamp` virtual column only has second precision, which is why `_timestamp_ms` with millisecond precision was added a couple of years ago too (https://github.com/ClickHouse/ClickHouse/pull/11260). All our metrics (such as `clickhouse_lag()`) using `_timestamp` can be off by up to 999 ms – better to use `_timestamp_ms` in new ones.

Reply:
Actually don't mind me, [discussed this with Yakko and seconds are good enough at the moment](https://posthog.slack.com/archives/C01MM7VT7MG/p1654184785125149)